storybook/pages/CommunitiesPortalLayoutPage.qml

```diff
@@ -10,10 +10,39 @@ import Storybook 1.0
 import Models 1.0
 
 import utils 1.0
+import mainui 1.0
 
 SplitView {
+    id: root
     Logs { id: logs }
 
+    Popups {
+        id: popups
+        appMain: root
+        rootStore: QtObject {}
+
+        Component.onCompleted: {
+            Global.openSendIDRequestPopup.connect(openSendIDRequestPopup)
+            Global.openOutgoingIDRequestPopup.connect(openOutgoingIDRequestPopup)
+            Global.openIncomingIDRequestPopup.connect(openIncomingIDRequestPopup)
+            Global.openInviteFriendsToCommunityPopup.connect(openInviteFriendsToCommunityPopup)
+            Global.openContactRequestPopup.connect(openContactRequestPopup)
+            Global.openChooseBrowserPopup.connect(openChooseBrowserPopup)
+            Global.openDownloadModalRequested.connect(openDownloadModal)
+            Global.openImagePopup.connect(openImagePopup)
+            Global.openProfilePopupRequested.connect(openProfilePopup)
+            Global.openNicknamePopupRequested.connect(openNicknamePopup)
+            Global.blockContactRequested.connect(openBlockContactPopup)
+            Global.unblockContactRequested.connect(openUnblockContactPopup)
+            Global.openChangeProfilePicPopup.connect(openChangeProfilePicPopup)
+            Global.openBackUpSeedPopup.connect(openBackUpSeedPopup)
+            Global.openEditDisplayNamePopup.connect(openEditDisplayNamePopup)
+            Global.openPinnedMessagesPopupRequested.connect(openPinnedMessagesPopup)
+            Global.openCommunityProfilePopupRequested.connect(openCommunityProfilePopup)
+            Global.openPopupRequested.connect(openPopup)
+        }
```

Comment:
We have this code repeated 1:1 in several places. I wonder if we would lose generality by moving those `connect` calls into `Popups` component directly to avoid repetitions. Or maybe we could create some kind of wrapper for `Popups` components responsible for connections. Wdyt?

Reply:
Maybe instead of exposing methods from `Popups` we could put `Connections` component into `Popups` and handle signal from Global internally, even without exposing methods for opening specific popup? Do we have some use cases for calling Popup's methods directly?

Reply:
Yeah, I don't like this too much either; will think about how to streamline this

Reply:
Done, moved directly to Popups